service/src/main/resources/logback.xml

```diff
@@ -1,14 +1,30 @@
 <configuration>
-    <conversionRule conversionWord="coloredLevel" converterClass="play.api.libs.logback.ColoredLevel" />
-
-    <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
-        <encoder>
-            <pattern>%date{HH:mm:ss.SSS} %highlight([%-5level]) %cyan(%logger{16}) [%thread] - %message%n%xException{10}</pattern>
+    <appender name="JSON-STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+        <encoder class="net.logstash.logback.encoder.LoggingEventCompositeJsonEncoder">
+            <providers>
+                <timestamp>
+                    <fieldName>timestamp</fieldName>
```

Comment:
not needed.

Reply:
actually needed to remove the output name with `@timestamp` -> `timestamp`
https://github.com/logstash/logstash-logback-encoder#standard-fields